zilliqa/src/consensus.rs

```diff
@@ -1088,26 +1087,22 @@ impl Consensus {
             debug!("ignoring new view from unknown node (buffer?) - committee size is : {:?} hash is: {:?} high hash is: {:?}", committee.len(), new_view.qc.block_hash, self.high_qc.block_hash);
             return Ok(None);
         };
+
+        let index = index as u16;
```

Comment:
this will wrap if (and I admit this is unlikely!) someone ever has an index > 65535. Suggest try_into() ..